compiler/src/dmd/iasmaarch64.d

```diff
@@ -0,0 +1,69 @@
+/**
+ * Inline assembler implementation for DMD.
+ * https://dlang.org/spec/iasm.html
+ *
+ * Copyright:   Copyright (C) 2025 by The D Language Foundation, All Rights Reserved
+ * Authors:     Walter Bright)
```

Comment:
```
src/dmd/iasmaarch64.d(13): Warning: Ddoc: Stray ')'. This may cause incorrect Ddoc output. Use $(RPAREN) instead for unpaired right parentheses.
module dmd.iasmaarch64;
```
I think it is compiling about the stray `)` at EOL here

Reply:
Good catch!